### PR TITLE
Should explain where to run the npm install

### PR DESCRIPTION
**Pull request by dugwood**
## Type of change
- [X] Other

## Description of change
It's not obvious for everyone that a git clone must be done first (especially if it's your first time with NodeJS!).

### CODE REVIEW

**Comment by headerbidding** [comment]
Yes, I have never used NodeJS before and have not been able to figure out how to compile my own prebid.js.

Perhaps a YouTube video showing how to do it would be useful.

(I need GA.  Or just include GA as an option for the quick download)

**Comment by dugwood** [comment]
In fact there's the whole code lower on the same page... So maybe the good idea is to remove the «Install» paragraph?

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ This README is for developers who want to contribute to Prebid.js.  For user-fac
 
 ## Install
 
+    $ git clone --depth=1 https://github.com/prebid/Prebid.js.git
```
Comment (matthewlane) [inline README.md]: Please omit the `--depth=1` portion Not a bad idea but standard `git clone` should suffice for an intro section

**Comment by dugwood** [comment]
It should be good now. How can I ask for the PR again?

**Comment by matthewlane** [comment]
Got it, merged

**Pull request merged**